dbt/models/location/location.vw_pin10_location_fill.sql

```diff
@@ -1,17 +1,5 @@
 -- View containing each of the PIN-level location (spatial joins)
 
--- Airport noise is joined differently since it's filled during ingest and
--- values for env_airport_noise_data_year won't match values for year
-WITH cyf_airport_noise AS (
-    SELECT
-        year,
-        CASE
-            WHEN env_airport_noise_data_year = 'omp' THEN year ELSE
-                env_airport_noise_data_year
-        END AS env_airport_noise_data_year
-    FROM {{ ref('location.crosswalk_year_fill') }}
-)
```

Comment:
We're not joining on `data_year` anymore, so this transformation only affects output columns and not join columns. As such, I moved it to a `CASE WHEN` statement in the `SELECT` clause of the query, which I think is simpler to read.